fix.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -x -e
+
+# Remove all fields from FcFontMap. Its' parent_instance field is broken and we don't need the type anyway.
+# Replace cairo_font_type_t with enums::FontType as well.
+xmlstarlet ed -P \
+	-d '//_:class[@name="FcFontMap"]/_:field' \
+	-u '//*[@c:type="cairo_font_type_t"]/@c:type' -v enums::FontType \
+	PangoCairo-1.0.gir > tmp
+mv tmp PangoCairo-1.0.gir
+
+# Remove Int32 alias because it misses c:type, it not like anyone actually cares about it.
+xmlstarlet ed -P \
+	-d '//_:alias[@name="Int32"]' freetype2-2.0.gir > tmp
```

Comment:
Strange, but this not works for me.
After `./dl.sh && ./fix.sh`

```diff
diff --git a/freetype2-2.0.gir b/freetype2-2.0.gir
index 7f06abc..afa2e5d 100644
--- a/freetype2-2.0.gir
+++ b/freetype2-2.0.gir
@@ -4,7 +4,9 @@
     <record name="Bitmap" c:type="FT_Bitmap"/>
     <record name="Face" c:type="FT_Face"/>
     <record name="Library" c:type="FT_Library"/>
-    
+    <alias name="Int32" c:type="FT_Int32">
+      <type name="int32"/>
+    </alias>
     <function name="library_version" c:identifier="FT_Library_Version">
       <return-value transfer-ownership="none">
         <type name="none" c:type="void"/>
```

```
$ xmlstarlet --version
1.6.1
compiled against libxml2 2.9.2, linked with 20907
compiled against libxslt 1.1.28, linked with 10132
```

Reply:
I retested scripts to make sure I got it right, but it does in fact work for
me. I am quite puzzled by your problem, because we apparently have exactly the
same version of xmlstarlet (though, libs are a little bit different):

```
$ xmlstarlet --version
1.6.1
compiled against libxml2 2.9.4, linked with 20904
compiled against libxslt 1.1.29, linked with 10129
```

The most likely root cause is related to namespaces as described here
http://xmlstar.sourceforge.net/doc/UG/ch05.html#idm47989546158480
I would appreciate if you could take a look and try one of suggestion there,
for example with explicit namespace binding.

Reply:
This not works too
```
xmlstarlet ed -P -N x="http://www.gtk.org/introspection/core/1.0" -d "//x:alias" freetype2-2.0.gir
```
For this works fine  `'///_:alias[@name="Int32"]'` with or without `-N`, don't understand why.
`xmlstarlet ed -P -d "//_:alias/_:type" freetype2-2.0.gir` works on deleting type.

Reply:
Other than namespaces, I am pretty sure that those expressions used in
xmlstarlet are correct. Given that binding namespaces didn't help, 
I am leaving those as is. 

Maybe it is problem in libxml2? It seems to have recently received attention of
security researchers and there is constant stream of various security fixes,
and fixes to regression caused by those fixes, and so on, and on ...